src/Microsoft.Health.Fhir.Tests.Common/TestFiles/Normative/SearchParameter.json

```diff
@@ -0,0 +1,41 @@
+{
+    "resourceType": "SearchParameter",
```

Comment:
>SearchParameter [](http://example.com/codeflow?start=21&length=15)

do you need this file?

Reply:
**This file is not part of the GraphQL work**, while doing a merge with the main branch, github took the new files from others as my changes.